datawork.py

```diff
@@ -80,13 +72,15 @@ def loadData(filename="covid_data.csv"):
         conn.commit()
         return "Data loaded successfully"
 
+
 def populateVaccineData():
     conn=connect()
     cursor=conn.cursor()
-    cursor.execute('INSERT INTO vaccine_data SELECT NumberDosesReceived,ReportingCountry,Vaccine FROM raw_data')
+    cursor.execute('INSERT INTO vaccine_data SELECT MAX(NumberDosesReceived),ReportingCountry,Vaccine FROM raw_data GROUP BY reportingcountry, vaccine')
```

Comment:
the original query created multiple rows per vaccine type per country, giving unreliable data. instead, this groups by country and vaccine type- creating a row with the maximum number of doses (probably the most recent)

Reply:
The original data has rows indexed by the week/year.
The intent of the original query is to make a copy of the original data but with only the important columns.
Then the code will add up the total doses per vaccine type per country.

We do not want to use the MAX(NumberofDosesReceived) here because it will not sum across all weeks.